### PR TITLE
Features/kb extension

### PR DESCRIPTION
**Pull request by marcoooo**
Extends Known bugs report to enable display / filter / export for Ensembl Release Intentions

### CODE REVIEW

File: production_services/settings.py [hunk 1]
```diff
@@ -100,7 +101,7 @@
         'ENGINE': 'django.db.backends.mysql',
         'NAME': os.getenv('USER_DB_DATABASE', 'ensembl_production_services'),
         'USER': os.getenv('USER_DB_USER', 'ensembl'),
-        'PASSWORD': os.getenv('USER_DB_PASSWORD', ''),
+        'PASSWORD': os.getenv('USER_DB_PASSWORD', 'ensembl'),
```
Comment (muffato) [inline production_services/settings.py:104]: Is it what it looks like ? You know `ensembl-production-services` is a public repository, right ?

Comment (marcoooo) [reply production_services/settings.py]: Yes I know.

**Comment by muffato** [comment]
(I'll remove myself from the reviewers, I'm not really qualified to review this code. I was just interested in seeing the new Known Bugs interface because http://www.ensembl.info/known-bugs/ doesn't work, and saw those strings)

**Comment by marcoooo** [comment]
> (I'll remove myself from the reviewers, I'm not really qualified to review this code. I was just interested in seeing the new Known Bugs interface because http://www.ensembl.info/known-bugs/ doesn't work, and saw those strings)

try http://www.ensembl.info/known-bugs/ensembl-100/ :-)

**Review by vinay-ebi — APPROVED** [review]

**Pull request merged**